catalog/dags/elasticsearch_cluster/create_proportional_by_source_staging_index/create_proportional_by_source_staging_index_dag.py

```diff
@@ -0,0 +1,188 @@
+"""
+# Create Proportional By Source Staging Index DAG
+
+This DAG is used to create a new staging Elasticsearch index that is a subset
+of a staging source index, such that the proportions of records by source in
+the new index is equal to the proportions of records by source in the source
+index.
+
+
+Required Dagrun Configuration parameters:
+
+* media_type:         The media type for which to create a new index.
+* percentage_of_prod: A float indicating the proportion of items to take from each
+                      source from the total amount existing in the staging
+                      source index
+
+Optional params:
+
+* source_index: An existing staging Elasticsearch index to use as the basis for
+                the new index. If not provided, the index aliased to
+                `<media_type>-filtered` will be used.
```

Comment:
Honestly, given how clearly this information is conveyed on the Trigger DAG page, I think it's okay to remove it from the DAG description and de-duplicate the places we're referencing it!
![image](https://github.com/WordPress/openverse/assets/10214785/f221d0cb-36c0-42f3-9966-e673ff7df409)

Reply:
I'm of two minds on this. I agree that the duplication is annoying, but (maybe not in this case specifically, but especially for some of the more complicated DAGs) it can be difficult to convey how a DAG works and what its available options are without just describing the params. Ideally, I think that information _should_ be conveyed from DAGs.md. It would be really nice if we could include this programmatically rather than duplicating the strings though 😓

Reply:
That's fair, we can leave it for now then 🙂